modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,39 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-understanding-intel-hardware-accelertor-card-for-openshift-container-platform_{context}"]
```

Comment:
Please match the ID to the file name as described in the second item of our [module file metadata](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata) section of the doc guidelines.